Unix/configure

```diff
@@ -2468,13 +2476,20 @@ OMI_CONFIGURE_QUALS=--enable-microsoft --disable-makefile-gen
 
 all:
 ifeq (\$(ENABLE_ULINUX),1)
+EOF
```

Comment:
I prefer you not do the EOF thing to write a fragment of the GNUmakefile file. This will make it harder to change GNUmakefile down the road. Just write the variables out (like they currently are, for ENABLE_ULINUX), and then test against them.